### PR TITLE
Shrike primordial: gravity grenade

### PR DESCRIPTION
**Pull request by TiviPlus**
Throw a gravity grenade thats sucks in everyone and everything in a 3 tile radius

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm [hunk 1]
```diff
@@ -328,3 +328,43 @@
...
+/obj/item/explosive/grenade/gravity
+	name = "gravity grenade"
+	desc = "A fleshy mass that seems way too heavy for its size. It seems to be vibrating."
+	icon_state = "alien_grenade"
+	det_time = 20
+
+/obj/item/explosive/grenade/gravity/prime()
+	new /obj/effect/overlay/temp/emp_pulse(loc)
+	playsound(loc, 'sound/effects/EMPulse.ogg', 50)
+	for(var/atom/movable/victim in view(3))//yes this throws EVERYONE
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm:362]: ditto on the GLOB.living_mob

Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm:362]: also not a huge fan of abilities displacing xenos like that, but we'll see

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm:362]: view != range

**Comment by Losenis** [comment]
Tell me this triggers impact nades if it hits an unga

**Review by BraveMole — APPROVED** [review]
Video of the ability?

**Review by Kuro020 — APPROVED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**